src/RuleSet/Sets/PSR12Set.php

```diff
@@ -29,6 +29,7 @@ public function getRules(): array
             'braces' => [
                 'allow_single_line_anonymous_class_with_empty_body' => true,
             ],
+            'class_definition' => ['space_before_parenthesis' => true],
```

Comment:
can we add a comment like that for this rule?
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/ControlStructure/ElseifFixer.php#L25